### ISSUE TITLE
Separate transfer (withdraw) and deposits tx

### ISSUE THREAD

**Issue by Bitcoinera** [DistributedCollective/FastBTC#28]
I am seeing that both the withdraw and deposit transactions are being stored in the same table, transactions.

This doesn't seem to be right, and rather leading to great confusion. It would be better to have these two separate, since right now the only thing to tell them apart is the txId being null or not. But doesn't seem like a good idea to keep them mixed in the same table

**Comment by Bitcoinera** [comment]
OK, now I see "type"... Still, I don't know if this is the best. I still find it confusing, having a field that is not common for both (see issue #29 )

**Comment by jamiemadrox779** [comment]
yes, you can split it into deposits/transfers and please  add a column which references deposit with transfer
